packages/components/src/CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing components
+
+These guidelines document how to contribute to components in the [`@wordpress/components`](https://www.npmjs.com/package/@wordpress/components) npm package, or “WordPress Components”. This can be addition, modification, or deprecation.
+
+## Does it belong in the component library?
+
+A component library should include what’s shared across many products and omit what’s not. Questions to help determine if a component should be added:
+
+- Is it relevant to other products/plugins?
+- Is there overlap with any existing components?
+- How much will it cost to make and maintain?
+- Can you articulate how the component should be used?
+
+Here’s a flowchart that can help determine if a new component is necessary:
+
+[![New component flowchart](https://wordpress.org/gutenberg/files/2019/07/New_component_flowchart.png)](https://coggle.it/diagram/WtUSrld3uAYZHsn-/t/new-ui-component/992b38cbe685d897b4aec6d0dd93cc4b47c06e0d4484eeb0d7d9a47fb2c48d94)
```

Comment:
I love a good flowchart, but it might be worth rebuilding this so it's a bit easier to read and uses some of WordPress' visual styles. It's a bit hard to read, and the horizontal alignment is going to make it hostile to users on mobile devices. (I'm happy to help here or offer more specific guidance as needed.)

Reply:
Thanks Sara, I think that sounds great. I'll consider it a non-blocking issue.